alphacsc/encoder.py

```diff
@@ -0,0 +1,196 @@
+import numpy as np
```

Comment:
Rename the file `_encoder.py` to make it clear it is private API.

Reply:
OK, done